src-php/Fields/CloudinaryImage.php

```diff
@@ -40,6 +40,6 @@ public function __construct($name, $attribute = null, $disk = 'cloudinary', $sto
             $path = pathinfo($model->{$this->attribute});
             Storage::disk($this->disk)->delete($path['filename']);
             return $this->columnsThatShouldBeDeleted();
-        });
+        })->help('Ensure when transformed to the size displayed on site, the image is less than the max file size for your Cloudinary account.');
```

Comment:
Could you please run the copy through the localisation to ensure it can easily be customised per implementation or for translation.

Reply:
Good shout!